lobbyists.mk

```diff
@@ -1,7 +1,17 @@
 # Lobbyist expenditures and contributions
-.PRECIOUS : data/intermediate/lobbyist_contributions.csv data/intermediate/lobbyist_expenditures.csv
+LOBBYIST_DATA_DIR=data/lobbyist
```

Comment:
This file is largely unchanged, apart from prefixing the targets with the data directory and reordering some recipes.